examples/ios/objc/Draw/README.md

```diff
@@ -0,0 +1,12 @@
+# Realm Draw Demo
```

Comment:
All other examples have their details in `examples/ios/objc/README.md`. Can you move this there for consistency, please?